PullRequestSandbox/Program.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
Ouch, it doesn't like that much does it? Is the preview pane being put off the page or something? :-/